Cargo.toml

```diff
@@ -40,6 +40,12 @@ bindgen = "0.66.1"
 llvm-tools = { version = "0.1.1", optional = true }
 
 [features]
-default = ["llvm-objcopy"]
+default = ["nrf9160", "llvm-objcopy"]
 arm-none-eabi-objcopy = []
 llvm-objcopy = ["dep:llvm-tools"]
+
+nrf9160 = []
+nrf9151 = ["nrf9120"]
+nrf9161 = ["nrf9120"]
+nrf9120 = []
```

Comment:
Could you mention these in the readme too so people know it supports the multiple versions?